From a1b8ff570b5ed95697f4748cfe3bbe154802c37e Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Sat, 2 Jul 2022 05:21:52 +0900
Subject: [PATCH] test: add a simple test for daemon-reexec

(cherry picked from commit d1b1bbfbfa1cb7d225250fc08089d1de17eaef7c)

Related: #2087652
---
 test/units/testsuite-03.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/units/testsuite-03.sh b/test/units/testsuite-03.sh
index 070e978cda..7c5a3b8f19 100755
--- a/test/units/testsuite-03.sh
+++ b/test/units/testsuite-03.sh
@@ -3,6 +3,10 @@
 set -eux
 set -o pipefail
 
+# Simple test for that daemon-reexec works in container.
+# See: https://github.com/systemd/systemd/pull/23883
+systemctl daemon-reexec
+
 # Test merging of a --job-mode=ignore-dependencies job into a previously
 # installed job.
 
